bin/pkg/search.ml

```diff
@@ -0,0 +1,114 @@
+open Import
+module Opam_repo = Dune_pkg.Opam_repo
+module Lock_dir = Dune_pkg.Lock_dir
+module Rev_store = Dune_pkg.Rev_store
+module Resolved_package = Dune_pkg.Resolved_package
+
+let get_default_lock_dir_path () = Dune_rules.Lock_dir.default_source_path |> Path.source
```

Comment:
I would suggest not doing that, as the lock dirs in source are mostly going away in #11775.

Better use `default_path` and run the build system to create it if it doesn't exist.

That way it will continue to work even if the lock dir was automatically generated by Dune.